README.md

```diff
@@ -55,20 +55,19 @@ A PythonAnywhere Account (for deploying the project on a live system)
 ---
 
 
-<details><summary>Step 2 : Installing Python</summary>
+<details><summary>Step 2 : Installing Python 3.7</summary>
  
  
-* Click on [Download Python](https://www.python.org/downloads/windows/).
-* Click on the "Latest Python 3 Release - Python x.x.x" link.
+* **For Windows** :  Click on [Download Python3.7](https://www.python.org/downloads/release/python-379/).
    * * Download the Windows x86-64 executable installer for 64-bit version of Windows
    * * Download the Windows x86 executable installer for 32-bit version of Windows.
 
 
-* Make sure to check "Add Python 3.x to Path" in the setup window of the Installer.
+   * * Make sure to check "Add Python 3.7 to Path" in the setup window of the Installer.
 
 Verify the installation from the command prompt using following command :
 ```
-python --version
+python3.7 --version
```

Comment:
Added link to download the python3.7 as original project has been made in python3.7. So, that existing requirements.txt file can work fine.